src/QueryBuilder/AbstractDDLQueryBuilder.php

```diff
@@ -169,14 +169,15 @@ public function createTable(string $table, array $columns, string $options = nul
     {
         $cols = [];
 
-        /** @psalm-var string[] $columns */
+        /** @psalm-var array<string, ColumnInterface>|string[] $columns */
         foreach ($columns as $name => $type) {
             if (is_string($name)) {
                 $cols[] = "\t"
                     . $this->quoter->quoteColumnName($name)
                     . ' '
                     . $this->queryBuilder->getColumnType($type);
             } else {
+                /** @psalm-var string $type */
                 $cols[] = "\t" . $type;
```

Comment:
```suggestion
                $cols[] = "\t" . $type;
```

if `$name` is integer, then here will throw exception, becuase `ColumnInterface` is not stringable.

Reply:
`ColumnInterface` can be only with string `$name`